src/types/Score.ts

```diff
@@ -0,0 +1,35 @@
+import strings from 'src/strings';
+
+// This will all be updated with real types when the BE is done
+
+export type Score = {
```

Comment:
In the backend, we had a more nested approach for organizing phase:

```
[
  {
    phase: "phase 0",
    scores: [
      {
        category: "Carbon",
        score: null | number
      }, ...
    ]
  },
  {
    phase: "phase 1",
    ...
  },
]
  
```

It may be easier to organize permissions going forward since scores should be frozen in previous phases but editable currently for example.

Reply:
Excellent, thanks for the info. I am going to wait until the BE is done before making too many modifications here as it is just for the purpose of moving the frontend at the moment.

Reply:
@tommylau523 I updated it a little bit to match the possible backend a bit closer.